src/stop

```diff
@@ -38,6 +38,13 @@ start_cmds.each do |key, cmd|
 
 end
 
+ensure_socket 'web' do
+  interface 'eth0'
+  port '22'
```

Comment:
I think you are right, for tcp/udp we'll have to use the forward port they configure